app/controllers/api/users/traces_controller.rb

```diff
@@ -7,7 +7,10 @@ class TracesController < ApiController
 
       def index
         @traces = current_user.traces.reload
-        render :content_type => "application/xml"
+        respond_to do |format|
+          format.xml { render :content_type => "application/xml" }
+          format.json { render :content_type => "application/json" }
+        end
```

Comment:
Same goes here.

Reply:
Done in: 42e841c31dc9b006b0fe1a6ba7eaa4413b9da5f3

Reply:
Currently, test cases seem to be failing with `Expected response to be a <2XX: success>, but was a <406: Not Acceptable>`

I think we would need something like  `before_action :set_request_formats` to set XML as default format, similar to what we're doing for element endpoints (node/way/relation).